apis/object/v1beta1/doc.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2020 The Crossplane Authors.
```

Comment:
```suggestion
Copyright 2023 The Crossplane Authors.
```

some other places as well